terms.html

```diff
@@ -5,8 +5,11 @@
 <dl class="termlist">
   <dt><dfn data-lt="claims">claim</dfn></dt>
   <dd>
-An assertion made about a <a>subject</a>.
-  </dd>
+A digital representation of an assertion made about an <a>entity</a> by a specific party.
+The entity is called the <a>subject</a> of the claim. 
+The party is referred to as the author of the claim.
+The meaning of the claim (its semantics) is determined by its author.
```

Comment:
I don't have any issues with this PR (conceptually)... in fact, I wish we had put work into terminology a year ago (but the WG was embroiled in other things, and still is).

The editorial concerns with the PR include:

* We are introducing a new term -- `party` and that will need to be defined and differentiated from `issuer` and `author`.
* We are introducing a new term -- `author` and that will need to be defined and differentiated from `issuer`, `author`, and `party`.

IOW, we are introducing new terminology that is difficult to differentiate from existing terminology... and that will take time to discuss. That said, terminology is a non-normative thing in the specification, and so we don't have to decide on this in the next couple of weeks... it can be something we do during the Candidate Recommendation phase. If we do this, however, we should spend the time to make sure that every new term we introduce clarifies rather than creates yet another thing that readers have to be aware of.

My concrete change request here is to define "party" and "author", OR assert that the terms are so obvious that they do not need to be defined.

Reply:
processed and resolved.